src/txkube/_swagger.py

```diff
@@ -327,17 +328,26 @@ def pclass_field_for_type(self, required):
 
 
 
+def provider_invariant(interface):
+    """
+    :param zope.interface.Interface interface: An interface to require.
+
+    :return: A pyrsistent invariant which requires that values provide the
```

Comment:
Is `pyrsistent` spelled correctly here or should that be `persistent`?
If it's wrong, fix that, and commit .